src/libs/ajax.js

```diff
@@ -131,12 +131,54 @@ const Leonardo = signal => ({
   }
 })
 
+const workspaceId = getConfig().workspaceId || '97c7cccb-aaf8-424c-92cc-587ba49919b6'
+const containerId = getConfig().containerResourceId || '181aa2f8-f72f-46c9-a06d-bff1cfa1bbbb'
```

Comment:
Unless these ids are cleared for public use I'd suggest clearing the latter half of the `OR` statements